messages/es.json

```
@@ -101,33 +101,33 @@
   "promotions.promotion.elligibility.firstBuy.true": "Verdadero",
   "promotions.promotion.elligibility.firstBuy.false": "Falso",
   "promotions.promotion.elligibility.creditCardBin.label": "BIN de tarjeta de crédito",
-  "promotions.promotion.elligibility.creditCardBin.verb.any": "es cualquier entre",
+  "promotions.promotion.elligibility.creditCardBin.verb.any": "es cualquiera entre",
   "promotions.promotion.elligibility.creditCardBin.placeholder": "Número de BIN…",
   "promotions.promotion.elligibility.creditCardBin.error": "{binsCount, plural, one {BIN} other {BINs}} {bins} {binsCount, plural, one {debe} other {deben}} tener como mínimo 6 y máximo 9 dígitos numéricos",
-  "promotions.promotion.elligibility.cartProduct.label": "Produto en carrito",
+  "promotions.promotion.elligibility.cartProduct.label": "Produto en el carrito",
   "promotions.promotion.elligibility.cartProduct.verb.brand.any": "pertenece a alguna de estas marcas",
   "promotions.promotion.elligibility.cartProduct.verb.brand.not.any": "no pertenece a alguna de estas marcas",
   "promotions.promotion.elligibility.cartProduct.verb.category.any": "pertenece a alguna de estas categorías",
-  "promotions.promotion.elligibility.cartProduct.verb.category.not.any": "no pertenece a alguna de estas categorías",
+  "promotions.promotion.elligibility.cartProduct.verb.category.not.any": "no pertenece a ninguna de estas categorías",
   "promotions.promotion.elligibility.cartProduct.verb.collection.any": "pertenece a alguna de estas colecciones",
-  "promotions.promotion.elligibility.cartProduct.verb.collection.not.any": "no pertenece a alguna de estas colecciones",
-  "promotions.promotion.elligibility.cartProduct.verb.product.any": "es cualquier entre",
-  "promotions.promotion.elligibility.cartProduct.verb.product.not.any": "no es cualquier entre",
+  "promotions.promotion.elligibility.cartProduct.verb.collection.not.any": "no pertenece a ninguna de estas colecciones",
+  "promotions.promotion.elligibility.cartProduct.verb.product.any": "es cualquiera entre",
+  "promotions.promotion.elligibility.cartProduct.verb.product.not.any": "no es cualquiera entre",
   "promotions.promotion.elligibility.cartProduct.verb.sku.any": "es cualquiera de estos SKUs",
   "promotions.promotion.elligibility.cartProduct.verb.sku.not.any": "no es cualquiera de estos SKUs",
   "promotions.promotion.elligibility.cartProduct.verb.itemPriceRange.gt": "con precio más alto que",
   "promotions.promotion.elligibility.cartProduct.verb.itemPriceRange.lt": "con precio inferior a",
   "promotions.promotion.elligibility.cartProduct.verb.itemPriceRange.between": "con precio entre",
   "promotions.promotion.elligibility.cartProduct.verb.fromToPrices": "con precio 'de'/'por'",
-  "promotions.promotion.elligibility.cartProduct.verb.seller.any": "es de uno de esos sellers",
+  "promotions.promotion.elligibility.cartProduct.verb.seller.any": "es de uno de esos vendedores",
```

Actually `Sellers` is an internal term known in our platform, thats why we left this in english (should be upper-cased tho)
```suggestion
  "promotions.promotion.elligibility.cartProduct.verb.seller.any": "es de uno de esos Sellers",
```